src/core/config/link_raw.h

```diff
@@ -78,7 +78,17 @@
  *
  */
 #ifndef OPENTHREAD_CONFIG_SOFTWARE_CSMA_BACKOFF_ENABLE
-#define OPENTHREAD_CONFIG_SOFTWARE_CSMA_BACKOFF_ENABLE 0
+#define OPENTHREAD_CONFIG_SOFTWARE_CSMA_BACKOFF_ENABLE 1
```

Comment:
is it possible to keep the original value?

Reply:
This was mistakenly changed. Will change it back.